Reference/using-ioc.md

```diff
@@ -156,12 +156,9 @@ Leave the UnityConfig file as is.
 
 Now create the following files to configure Unity correctly for an Umbraco site.
 
-	class UnityEvents : IApplicationEventHandler
+	public class UnityEvents : ApplicationEventHandler
```

Comment:
This is incorrect, there is a reason why it is using the interface! If you use the base class, then you need to override both: `ExecuteWhenApplicationNotConfigured` and `ExecuteWhenDatabaseNotConfigured` and return true for both of these, by default this is false because this base class is generally just used for plugins/extensions but your container must always be populated.